samples/certs/README.md

```diff
@@ -24,6 +24,8 @@ The following `make` targets are defined:
 
 - `make root-ca`: this will generate a new root CA key and certificate.
 - `make $NAME-certs`: this will generate all needed files to bootstrap a new Citadel for cluster `$NAME` (e.g., `us-east`, `cluster01`, etc.).
+- `make $NAME-certs-wl`: this will generate certificates for a virtual machine connected to the namespace `$NAMESPACE` using
```

Comment:
NAME-certs: this will generate certificate for NAME namespace, in the NAME/ directory. 

I think we can skip the NAMESPACE thing.

Reply:
Maybe add info about "for 'default' service account. To use a different service account, define xxx"